assets/js/algolia-search.js

```diff
@@ -3,8 +3,14 @@ const searchClient = algoliasearch(
   "0ba2c5f100ff5fd004415e4abbcf9b9c"
 );
 
+const PROD_URL = "https://www.egazette.gov.sg";
+const isProd = window.location.origin === PROD_URL;
+// const algoliaIndexName = isProd
+//   ? "ogp_egazettes_index"
+//   : "staging_ogp_egazettes_index";
+const algoliaIndexName = "test_snippet_1";
```

Comment:
this should be fixed before merge

if this is a test pr, let's mark it as a draft pr

Reply:
ya converting to draft, the main data set is not ready for filtering